src/authentication/authentication.service.ts

```diff
@@ -31,7 +31,7 @@ export class AuthenticationService {
     id: number,
     updatePhoneNumberData: UpdatePhoneNumberDto,
   ) {
-    return await this.usersService.updatePhoneNumber(id, updatePhoneNumberData);
+    return this.usersService.updatePhoneNumber(id, updatePhoneNumberData);
   }
```

Comment:
This function is redundant, such funciton already exists in the usersService, so you can call `usersService.updatePhoneNumber` in the controller